lib/find.coffee

```diff
@@ -117,6 +117,20 @@ module.exports =
       'find-and-replace:select-skip': (event) ->
         selectNextObjectForEditorElement(this).skipCurrentSelection()
 
+    maybeRegisterToAutocompletePlus = (atomPackage) =>
```

Comment:
> It would be better to expose a service from autocomplete plus that exposes the ability to observe providers and consume it in this package.

I might be missing something, but isn't that similar to this implementation, only that here the `ProviderManager` exposes the ability? Or would you like this to avoid the coupling with the `ProviderManager`?

About hard-coding the support, that definitely makes sense too! I only thought it might be beneficial to have a recipe for adding autocompletion to X, where X might be a third-party package.